src/lib/y2storage/boot_requirements_strategies/zipl.rb

```diff
@@ -48,6 +46,17 @@ def errors
         errors
       end
 
+      def fatal_errors
+        res = super
+        return res unless res.empty?
+
+        if missing_partition_for?(zipl_volume)
```

Comment:
Why to check with a volume and report the error with another?

Reply:
@joseivanlopez bug...btw in bootloader check is that /boot/zip *path* is on ext4. So we are fine if /boot is ext4 and no separate /boot/zipl needed or even / as ext4. how I can express it with volume?